t/01_ps.t

```diff
@@ -1,18 +1,17 @@
 use strict;
 use warnings;
+use FindBin;
 use File::Spec;    # try to keep pathnames neutral
 use Test::More 0.96;
 
-BEGIN { use_ok("HTML::FormatPS"); }
+use lib "$FindBin::Bin/lib";
```

Comment:
The flipside is that if you have a hefty test directory structure, it's nice to be able to run just part of the tests by saying cd t/complex/directory/hierarchy and then either prove *.t or even perl -d that-one-test-that-fails.t and not have to prefix everything with t/complex/directory/hierarchy all the time.

(I'm from a RISC OS background, where making assumptions about the current working directory is one of the greatest sins. I fear I'm fighting a losing battle in Linux-land, though, given that e.g. git blame /path/to/repository/file fails if you're not actually currently inside /path/to/repository yourself, for reasons I don't understand but are presumably due to kernel developers thinking "why would you ever want to do that?")

But I'll fix, in the interests of getting this resolved, and because it's not a major win for such a small test suite like this.